### PR TITLE
CVE-2021-33203

### PR DESCRIPTION
**Pull request by rickprice**

### CODE REVIEW

File: docs/releases/1.11.29.1.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+``TemplateDetailView`` view to check the existence of arbitrary files.
+Additionally, if (and only if) the default admindocs templates have been
+customized by the developers to also expose the file contents, then not only
+the existence but also the file contents would have been exposed.
+
+As a mitigation, path sanitation is now applied and only files within the
+template root directories can be loaded.
+
+
+*March 4, 2020*
```
Comment (ucodery) [inline docs/releases/1.11.29.1.txt:22]: This already exists in the 1.11.29.txt release note and doesn't need repeating.

File: docs/releases/1.11.29.1.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+============================
+Django 1.11.29 release notes
```
Comment (ucodery) [inline docs/releases/1.11.29.1.txt:2]: Both of these version numbers need corrected.

**Review by ucodery — CHANGES_REQUESTED** [review]

File: docs/releases/1.11.29.1.txt (hunk 1 above)
Comment (rickprice) [reply docs/releases/1.11.29.1.txt]: Yeah, that was an oops.

File: docs/releases/1.11.29.1.txt (hunk 2 above)
Comment (rickprice) [reply docs/releases/1.11.29.1.txt]: Changed.

**Review by rickprice — COMMENTED** [review]
I made the requested changes, not sure how to show that, but they are made now.

**Review by ucodery — APPROVED** [review]

**Pull request merged**